modules/lambda-registrator/variables.tf

```diff
@@ -32,20 +30,20 @@ variable "enterprise" {
   default     = false
 }
 
-variable "partitions" {
-  description = "Specifies the partitions that Lambda registrator will manage [Consul Enterprise]."
+variable "admin_partitions" {
+  description = "Specifies the admin partitions that Lambda function registrator will manage [Consul Enterprise]."
   type        = list(string)
-  default     = []
+  default     = var.enterprise ? [] : null
 }
 
 variable "timeout" {
-  description = "The maximum number of seconds Lambda registrator can run before timing out."
+  description = "The maximum number of seconds Lambda function registrator can run before timing out."
```

Comment:
I think "Lambda registrator" is clear enough and often the functions are just referred to as "lambdas" and so I don't see "function" as helping much, just makes it longer. And then if we keep the name "Lambda registrator" then I think it would be confusing to rename it in these comments.

Reply:
>I think "Lambda registrator" is clear enough and often the functions are just referred to as "lambdas" and so I don't see "function" as helping much, just makes it longer.

I think/prefer using the specific, precise terms of the service in documentation is greater than referring to the colloquialism of the service 😄 , but am happy to shelve this suggestion.

To add...

>often the functions are just referred to as "lambdas" and so I don't see "function" as helping much, just makes it longer.

Often in docs, HashiCorp/otherwise, I have observed a pattern of inconsistency arising from the assumption in which one group of practitioners understands a term or concept in one manner, that others will/already understand it in this unofficial manner, as agreed upon by the other group. While the experienced of us can observe the pattern here, that a reference to "lambda" maps to "lambda function," a practitioner may react in one of two ways:

* Wonder what the difference is between "lambdas" and "lambda functions?" Is there a difference?

* Wonder why we refer to the functions as Lambdas when AWS doesn't. Is there something that *I* don't know? 

In both circumstances, I would kindly make the argument that we create the opportunity for losing trust with the practitioner by not using consistent jurisprudence with how we name and refer to things that are not within our span-of-control. I understand the mountain I'm making here (and with no ill repute I promise! 😄 ) considering the size of the change I suggested (And am happy to resolve as uncommitted, as your review comment is correct without rebuttal), but it reminded me of the opportunities that we have in front of us to improve the practitioner experience that I wanted to relay to the group.

✨